GitPrompt.ps1

```diff
@@ -138,13 +138,22 @@ function Write-GitStatus($status) {
             }
             $repoName = Split-Path -Leaf (Split-Path $status.GitDir)
             $prefix = if ($s.EnableWindowTitle -is [string]) { $s.EnableWindowTitle } else { '' }
-            $Host.UI.RawUI.WindowTitle = "$prefix$repoName [$($status.Branch)]"
+            $adminHeader = Get-AdminTitleHeader

```

Comment:
Do you have any concerns about performance? I wonder if it would be sufficient to check admin-ness once when the module loads and use that for the rest of the session?

Or it may be that the security stuff is trivial in cost compared to the Git work we're doing?